R/run.R

```diff
@@ -441,10 +460,13 @@ global_setup <- function(lib_path = NULL, cpu_count = NULL, mem_alloc = NULL,
 #' @param name Benchmark name, i.e. `bm$name`
 #' @param params Named list of parameters for the individual run, i.e. the case
 #' @param cpu_count Number of CPUs allocated
+#' @param drop_caches Attempt to drop the disk cache before each case or iteration.
```

Comment:
Is it worth inheriting this from `measure`? Or the other way around?

Reply:
I ended up making the `measure` one a bit more precise, since the values have different implications there (i.e. `"iteration"` drops once, and both `"case"` and `NULL` result in no dropping)